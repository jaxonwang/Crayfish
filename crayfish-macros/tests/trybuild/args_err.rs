use crayfish_macros::*;

#[arg]
mod foo {}

pub fn main() {}
