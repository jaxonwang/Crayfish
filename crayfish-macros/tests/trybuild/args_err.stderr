error: attributes only applys to struct, enum.
 --> $DIR/args_err.rs:4:1
  |
4 | mod foo {}
  | ^^^^^^^^^^
